src/Source/Source.js

```diff
@@ -145,6 +146,7 @@ class Source extends InformationsData {
         this.isVectorSource = (source.parser || supportedParsers.get(source.format)) != undefined;
         this.networkOptions = source.networkOptions || { crossOrigin: 'anonymous' };
         this.attribution = source.attribution;
+        this.bboxUrlPrecision = source.bboxUrlPrecision === undefined ? 2 : source.bboxUrlPrecision;
```

Comment:
Following previous comments, you should move this assignment to `WMSSource` and `WFSSource` as follow:
```js
this.bboxUrlPrecision = source.bboxUrlPrecision;
```

Reply:
Done, I move code to WMSource and WFSSource